code-challenges.test.js

```diff
@@ -24,28 +31,83 @@
 
 
 // b) Create the function that makes the test pass.
-
-
+/*
+Algorithm->find an array that display the sequence of fibonacci numbers
+            Input any number greater than 2
+            Output->an array of fibonacci sequence of a given number
+                declare three variables num1,num2,nextNum
+                create a function 
+                create an empty array
+                if number less than 2 return number
+                else use for loop
+                nextNum=num1+num2
+                num1=num2
+                num2=nextNum
+                return sequence
+*/
+let num1=1,num2=1,nextNum
+let fibonacci=(number)=>{
+    let sequence=[]
+    if(number<2){return number
+    }else{
+        for(let i=0;i<number;i++){
+        sequence.push(num1)
+        nextNum=num1+num2
+        num1=num2
+        num2=nextNum              
+}
+    }return sequence
+}
```

Comment:
this is excellent work! If you want you can populate arrays with static data that you know they will use consistently.